.github/workflows/build_and_test_release_latest.yml

```diff
@@ -0,0 +1,69 @@
+name: Test utils_common
+on:
+  schedule:
+    # Run every hour. This helps detect flakiness,
+    # and broken external dependencies.
+    - cron:  '0 * * * *'
+
+jobs:
+  build_and_test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ros_distro: [kinetic, melodic, dashing]
+        include:
+        - ros_distro: kinetic
+          ubuntu_distro: xenial
+        - ros_distro: melodic
+          ubuntu_distro: bionic
+        - ros_distro: dashing
+          ubuntu_distro: bionic
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-${{ matrix.ubuntu_distro }}-ros-${{ matrix.ros_distro }}-ros-base-latest
+    outputs:
+      build_result: ${{ steps.build.outcome }}
+      test_result: ${{ steps.test.outcome }}
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
`ros-tooling/action-ros-ci` will completely disregard what you check out with `actions/checkout` here.

Reply:
Right, shoot.